### DISCUSSION TITLE
Problems with TODO Codespaces vscode to github issues

### DISCUSSION THREAD

**Discussion by dimaguy** [community/community#26850] [Codespaces]
<p>Hello everyone! This is my first day with Codespaces Beta and I am finding it to be very cool!<br>
Now onto business :stuck_out_tongue::</p>
<p>I have found a problem with TODO comments.</p>
<p><strong>Current Behaviour</strong><br>
When making a TODO comment</p>
<pre><code class="lang-auto">vb.net
' TODO:placeholder here
</code></pre>
<p>VSCode suggests creating a github issue<br>
<img src="https://user-images.githubusercontent.com/7207103/181100276-96b6aed5-7e68-4308-a445-75ced237b61a.png" alt="imagem" data-base62-sha1="80DELzkkvfllSMc2TWyTtd6pqTB" width="202" height="80"><br>
But it actually does not check if the repository is up to date (if the code in the repo matches the local branch) and makes it likely to point the snippet at another line that does not correspond to the TODO comment line that was supposed to be mentioned.</p>
<p><strong>How it was supposed to behave</strong></p>
<p>1-<em>click on “Create GitHub Issue”</em><br>
2-The issue opens pointing at the correct line of the TODO comment.</p>
<p><strong>Suggested Fixes</strong><br>
1- Quote the code instead of mirroring it from the file and point to which file it is in.<br>
or<br>
2- Not let the github issue be created unless local code matches repository.</p>

**Comment by lostintangent** [comment, marked as answer]
<p>Hey! This sounds like it’s likely an issue with the <a href="https://github.com/microsoft/vscode-pull-request-github" rel="noopener nofollow ugc">GitHub Pull Requests and Issues extension</a>, which Codespaces simply installs by default. Would it make sense to file an issue on that issue tracker?</p>

**Comment by dimaguy** [comment]
<p>Probably yes! Thanks for pointing me the right way</p>
<p>EDIT: Issue opened - <a href="https://github.com/microsoft/vscode-pull-request-github/issues/2166" rel="noopener nofollow ugc">https://github.com/microsoft/vscode-pull-request-github/issues/2166</a></p>
